src/main/java/bio/terra/cli/service/WorkspaceManagerServiceAws.java

```diff
@@ -88,58 +169,299 @@ public AwsS3StorageFolderResource createControlledAwsS3StorageFolder(
+    CreateControlledAwsSageMakerNotebookRequestBody createRequest =
+        new CreateControlledAwsSageMakerNotebookRequestBody()
+            .common(createCommonFields(createParams.resourceFields))
+            .awsSageMakerNotebook(fromCLIObject(createParams))
+            .jobControl(new JobControl().id(asyncJobId));
 
-    handleClientExceptions(
+    return handleClientExceptions(
         () -> {
           ControlledAwsResourceApi controlledAwsResourceApi =
               new ControlledAwsResourceApi(apiClient);
-          // make the initial delete request
+          // make the initial create request
           HttpUtils.callWithRetries(
-              () ->
-                  controlledAwsResourceApi.deleteAwsS3StorageFolder(
-                      deleteRequest, workspaceId, resourceId),
+              () -> controlledAwsResourceApi.createAwsSageMakerNotebook(createRequest, workspaceId),
               WorkspaceManagerService::isRetryable);
 
           // poll the result endpoint until the job is no longer RUNNING
-          DeleteControlledAwsResourceResult deleteResult =
+          CreateControlledAwsSageMakerNotebookResult createResult =
               HttpUtils.pollWithRetries(
                   () ->
-                      controlledAwsResourceApi.getDeleteAwsS3StorageFolderResult(
+                      controlledAwsResourceApi.getCreateAwsSageMakerNotebookResult(
                           workspaceId, asyncJobId),
                   (result) -> isDone(result.getJobReport()),
                   WorkspaceManagerService::isRetryable,
                   60,
                   Duration.ofSeconds(10));
-          logger.debug("delete controlled AWS S3 Storage Folder result: {}", deleteResult);
+          logger.debug("create controlled AWS SageMaker Notebook result: {}", createResult);
 
-          throwIfJobNotCompleted(deleteResult.getJobReport(), deleteResult.getErrorReport());
-          return true;
+          throwIfJobNotCompleted(createResult.getJobReport(), createResult.getErrorReport());
+          return createResult.getAwsSageMakerNotebook();
         },
-        "Error deleting controlled AWS S3 Storage Folder in the workspace.");
+        "Error creating controlled AWS SageMaker Notebook in the workspace.");
+  }
+
+  /**
+   * Call the Workspace Manager POST
+   * "/api/workspaces/v1/{workspaceId}/resources/controlled/aws/notebook/{resourceId}" endpoint to
+   * delete a AWS SageMaker Notebook as a controlled resource in the workspace.
+   *
+   * @param workspaceId the workspace to remove the resource from
+   * @param resourceId the resource id
+   * @throws SystemException if the job to delete the notebook fails
+   * @throws UserActionableException if the CLI times out waiting for the job to complete
+   */
+  public void deleteControlledAwsSageMakerNotebook(UUID workspaceId, UUID resourceId) {
+    deleteControlledAwsResource(workspaceId, resourceId);
```

Comment:
How does this delete the sagemaker notebook? `deleteControlledAwsResource` only calls storage folder delete. Or is this still wip